File: client/iih_stub/iih.ci_stub/iih.ci.ord/emsdi/d/OrWfDeptInfoDTO.cs
using System;
using xap.mw.core.data;
using xap.mw.coreitf.d;

namespace iih.ci.ord.emsdi.d
{
    /// <summary>
    /// OrWfDeptInfoDTO 的摘要说明。
    /// </summary>
    public class OrWfDeptInfoDTO : BaseDTO {

        public OrWfDeptInfoDTO() {
 
        }

        /// <summary>
        /// 医嘱流向执行科室列表
        /// </summary>
		public FArrayList Orwfexedepts {
            get { return getAttrVal<FArrayList>("Orwfexedepts",null); }
            set { setAttrVal<FArrayList>("Orwfexedepts", value); }
        }

        /// <summary>
        /// 物品流向库房科室列表
        /// </summary>
		public FArrayList Pharmwfwhdepts {
            get { return getAttrVal<FArrayList>("Pharmwfwhdepts",null); }
            set { setAttrVal<FArrayList>("Pharmwfwhdepts", value); }
        }

        /// <summary>
        /// 第一个执行科室id
        /// </summary>
		public string Firstid_mp_dept {
            get { return getAttrVal<string>("Firstid_mp_dept",null); }
            set { setAttrVal<string>("Firstid_mp_dept", value); }
        }

        /// <summary>
        /// 第一个执行科室名称
        /// </summary>
		public string Firstname_mp_dept {
            get { return getAttrVal<string>("Firstname_mp_dept",null); }
            set { setAttrVal<string>("Firstname_mp_dept", value); }
        }

        /// <summary>
        /// 执行科室字符串
        /// </summary>
		public string Id_mp_depts {
            get { return getAttrVal<string>("Id_mp_depts",null); }
            set { setAttrVal<string>("Id_mp_depts", value); }
        }

        /// <summary>
        /// 库存id
        /// </summary>
		public string Id_dept_wh {
            get { return getAttrVal<string>("Id_dept_wh",null); }
            set { setAttrVal<string>("Id_dept_wh", value); }
        }

        /// <summary>
        /// 库房名称
        /// </summary>
		public string Name_dept_wh {
            get { return getAttrVal<string>("Name_dept_wh",null); }
            set { setAttrVal<string>("Name_dept_wh", value); }
        }

        /// <summary>
        /// 库房主键字符串拼接
        /// </summary>
		public string Id_dept_whs {
            get { return getAttrVal<string>("Id_dept_whs",null); }
            set { setAttrVal<string>("Id_dept_whs", value); }
        }
        /// <summary>
        /// 返回属性列表
        /// </summary>
        /// <returns></returns>
        public override string[] getAttrNames()
        {
            return new string[]{ "Orwfexedepts", "Pharmwfwhdepts", "Firstid_mp_dept", "Firstname_mp_dept", "Id_mp_depts", "Id_dept_wh", "Name_dept_wh", "Id_dept_whs"};
        }
        
        /// <summary>
        /// 返回全路径DO类名
        /// </summary>
        /// <returns></returns>
        public override string getFullClassName()
        {
            return "iih.ci.ord.emsdi.d.OrWfDeptInfoDTO";
        }
    }
}

File: server/iih_pub/iih.ci_pub/src/main/java/iih/ci/ord/emsdi/d/OrWfDeptInfoDTO.java
package iih.ci.ord.emsdi.d;

import xap.mw.core.data.*;
import xap.mw.coreitf.d.*;
import java.math.BigDecimal;

/**
 * 医嘱流向科室数据信息DTO DTO数据 
 * 
 */
public class OrWfDeptInfoDTO extends BaseDTO {
	private static final long serialVersionUID = 1L;
	
	/**
	 * 医嘱流向执行科室列表
	 * @return FArrayList
	 */
	public FArrayList getOrwfexedepts() {
		return ((FArrayList) getAttrVal("Orwfexedepts"));
	}
	/**
	 * 医嘱流向执行科室列表
	 * @param Orwfexedepts
	 */
	public void setOrwfexedepts(FArrayList Orwfexedepts) {
		setAttrVal("Orwfexedepts", Orwfexedepts);
	}
	/**
	 * 物品流向库房科室列表
	 * @return FArrayList
	 */
	public FArrayList getPharmwfwhdepts() {
		return ((FArrayList) getAttrVal("Pharmwfwhdepts"));
	}
	/**
	 * 物品流向库房科室列表
	 * @param Pharmwfwhdepts
	 */
	public void setPharmwfwhdepts(FArrayList Pharmwfwhdepts) {
		setAttrVal("Pharmwfwhdepts", Pharmwfwhdepts);
	}
	/**
	 * 第一个执行科室id
	 * @return String
	 */
	public String getFirstid_mp_dept() {
		return ((String) getAttrVal("Firstid_mp_dept"));
	}
	/**
	 * 第一个执行科室id
	 * @param Firstid_mp_dept
	 */
	public void setFirstid_mp_dept(String Firstid_mp_dept) {
		setAttrVal("Firstid_mp_dept", Firstid_mp_dept);
	}
	/**
	 * 第一个执行科室名称
	 * @return String
	 */
	public String getFirstname_mp_dept() {
		return ((String) getAttrVal("Firstname_mp_dept"));
	}
	/**
	 * 第一个执行科室名称
	 * @param Firstname_mp_dept
	 */
	public void setFirstname_mp_dept(String Firstname_mp_dept) {
		setAttrVal("Firstname_mp_dept", Firstname_mp_dept);
	}
	/**
	 * 执行科室字符串
	 * @return String
	 */
	public String getId_mp_depts() {
		return ((String) getAttrVal("Id_mp_depts"));
	}
	/**
	 * 执行科室字符串
	 * @param Id_mp_depts
	 */
	public void setId_mp_depts(String Id_mp_depts) {
		setAttrVal("Id_mp_depts", Id_mp_depts);
	}
	/**
	 * 库存id
	 * @return String
	 */
	public String getId_dept_wh() {
		return ((String) getAttrVal("Id_dept_wh"));
	}
	/**
	 * 库存id
	 * @param Id_dept_wh
	 */
	public void setId_dept_wh(String Id_dept_wh) {
		setAttrVal("Id_dept_wh", Id_dept_wh);
	}
	/**
	 * 库房名称
	 * @return String
	 */
	public String getName_dept_wh() {
		return ((String) getAttrVal("Name_dept_wh"));
	}
	/**
	 * 库房名称
	 * @param Name_dept_wh
	 */
	public void setName_dept_wh(String Name_dept_wh) {
		setAttrVal("Name_dept_wh", Name_dept_wh);
	}
	/**
	 * 库房主键字符串拼接
	 * @return String
	 */
	public String getId_dept_whs() {
		return ((String) getAttrVal("Id_dept_whs"));
	}
	/**
	 * 库房主键字符串拼接
	 * @param Id_dept_whs
	 */
	public void setId_dept_whs(String Id_dept_whs) {
		setAttrVal("Id_dept_whs", Id_dept_whs);
	}
}